apps/greencheck/tests/views/test_stats_view.py

```diff
@@ -207,6 +210,7 @@ def test_stat_view_top_domains(
 
         assert len(top_green_domains) == 10
 
+    @pytest.mark.flaky
```

Comment:
This code isn't really used, and we don't expose the stats anywhere. I've commented out the tests for now, and it might make sense to remove it entirely in a separate PR if we're not exposing this to end users.